src/Compute/Compute.Autorest/UX/Microsoft.Compute/virtualMachineScaleSets-virtualMachines-runCommands.json

```diff
@@ -1,6 +1,6 @@
 {
   "resourceType": "virtualMachineScaleSets/virtualMachines/runCommands",
-  "apiVersion": "2021-07-01",
+  "apiVersion": "2023-07-01",
```

Comment:
Why not the latest version? 2024-03-01.

Reply:
the generated cmdlets have not migrated to the newest version yet as there has not been a need to. 

The readme files that are used to generated cmdlets still point to 2023-07-01, and these files are agnostic to the newer version